db/migrate/20230317114940_add_directive_academy_order_to_projects.rb

```diff
@@ -0,0 +1,14 @@
+class AddDirectiveAcademyOrderToProjects < ActiveRecord::Migration[7.0]
+  def up
+    add_column :projects, :directive_academy_order, :boolean, default: false
+
+    Project.all.each do |project|
+      project.directive_academy_order = !(project.task_list_type == "Conversion::Voluntary::TaskList")
```

Comment:
I actually hate this syntax but the linter is demanding it. I think it's quite obfuscated.

Reply:
Yeah, it's pretty hard to parse for me too! The only thing I could propose would be:

```
def directive_academy_required(project)
return false if project.task_list_type == "Conversion::Voluntary::TaskList"
return true if project.task_list_type == "Conversion::Inoluntary::TaskList"
end
```